src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -90,6 +90,36 @@ export type Procedure = {
 }
 
 export const NAME = 'meta'
+
+export type ClientSettings = {
+  allowOutgoingConnections: boolean
+  credentialTimeout: number | string // number (seconds) or duration string (support other units)
+  postConnectCmd: string
+  remoteContentHostnameAllowlist: string
+  retainConnectionCredentials: boolean
+  retainEditorHistory: boolean
+  allowTelemetry: boolean
+  authEnabled: boolean
+  metricsNamespacesEnabled: boolean
+  metricsPrefix: string
+}
+
+/**
+ * Initial client settings, used before the actual settings is loaded. Not to be
+ * confused with the default values for the setting, since not always the same.
+ */
+export const initialClientSettings: ClientSettings = {
+  allowOutgoingConnections: false, // default is true, but set to false until settings read
+  credentialTimeout: 0,
+  postConnectCmd: '',
+  remoteContentHostnameAllowlist: 'guides.neo4j.com, localhost',
+  retainConnectionCredentials: false, // default is true, but set to false until settings read
+  retainEditorHistory: false, // default is true, but set to false until settings read
+  allowTelemetry: true, // default is true. Renamed to client.allow_telemetry after 5.0
+  authEnabled: true, // default is true, but set to false until settings read
```

Comment:
This comment is outdated

Reply:
We could probably leave out the comments about renaming here as well since you have renaming comments further down :+1: